cosim/black-parrot-example/fpga/Makefile

```diff
@@ -32,11 +50,9 @@ include $(TOP)/cosim/mk/Makefile.fpga
 # Modify base flist
 #############################
 $(FLIST): $(BLACKPARROT_DIR)/bp_top/syn/flist.vcs $(BASE_FLIST)
-	echo "+incdir+$(CURR_SRC_DIR)" > $@
```

Comment:
Was there an issue with the previous code?

Reply:
I think the line was just redundant.

Reply:
Oh, where else is the v/ include given?